lib/rollbar/util.rb

```diff
@@ -66,9 +66,9 @@ def self.deep_copy(obj)
 
     def self.deep_merge(hash1, hash2)
       hash2.each_key do |k|
-        if hash1[k].is_a? ::Hash and hash2[k].is_a? ::Hash
+        if hash1[k] and hash1[k].is_a? ::Hash and hash2[k].is_a? ::Hash
```

Comment:
If we are actually checking that `hash1[k]` is a `Hash` object, why we need to check it exists? 

`nil.is_a?(Hash) # false`